ua-parser/1.3.0.wso2v2/pom.xml

```diff
@@ -0,0 +1,91 @@
+<!--
+ ~ Copyright (c) 2005-2010, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shouldn't year change?

Reply:
Fixed in https://github.com/wso2/orbit/pull/393